src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -42,7 +42,7 @@ public ConsoleLogger(string name, Func<string, LogLevel, bool> filter, bool incl
             Filter = filter ?? ((category, logLevel) => true);
             IncludeScopes = includeScopes;
 
-            if (RuntimeEnvironmentHelper.IsWindows)
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Why did you link the runtime info api as a build only dep on dnx451? The library is not being found at runtime. I might try adding it as a dep myself to see if stuff gets fixed.